spec/features/account_history_spec.rb

```diff
@@ -40,10 +59,17 @@
     expect(page).to have_link(
       identity_with_link.display_name, href: 'http://localhost:3000'
     )
+
```

Comment:
After reading this [thoughtbot example](https://thoughtbot.com/upcase/test-driven-rails-resources/rspec_acceptance.pdf), I thought about splitting the below expectations into a separate method called something like `user_sees_events_in_descending_order`. And moving all the above expectations into a separate method called something like `user_sees_event_history`. These changes would improve the test's readability, but I wasn't sure they were necessary/desired.